monkey/monkey_island/cc/ui/src/components/report-components/AttackReport.js

```diff
@@ -73,22 +76,28 @@ class AttackReport extends React.Component {
         return <FontAwesomeIcon icon={faEye} className={'technique-status-icon'}/>;
       case ScanStatus.USED:
         return <FontAwesomeIcon icon={faRadiation} className={'technique-status-icon'}/>;
-      default:
```

Comment:
We should keep default so that the report doesn't crash on other people's forks/tests of Monkey